source/2_platform/rex_opengl/src/opengl_renderer.cpp

```diff
@@ -18,6 +18,7 @@ namespace rex
   //-------------------------------------------------------------------------
   enum class ClearBits
   {
+    None               = 0,
```

Comment:
I would opt to make this a negative value, OpenGL might have a reserved place for 0 maybe not in this case but just to be consistent. Negative values are most certainly invalid though.